File: CS/Form1.cs
using System;
using System.Windows.Forms;

namespace ManageReportBands {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private XtraReport1 report = new XtraReport1();

        private void button1_Click(object sender, System.EventArgs e) {
            report.CreateDocument();
            report.ShowPreview();
        }

        private void button2_Click(object sender, System.EventArgs e) {
            report.AddMarginBands();
        }

        private void button3_Click(object sender, System.EventArgs e) {
            report.RemoveMarginBands();
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace ManageReportBands
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private report As New XtraReport1()

		Private Sub button1_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles button1.Click
			Dim printTool As New ReportPrintTool(report)
			CType(printTool.Report, XtraReport1).CreateDocument()
			CType(printTool.Report, XtraReport1).ShowPreviewDialog()
		End Sub

		Private Sub button2_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles button2.Click
			report.AddMarginBands()
		End Sub

		Private Sub button3_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles button3.Click
			report.RemoveMarginBands()
		End Sub
	End Class
End Namespace